docs/config.md

```diff
@@ -95,6 +95,45 @@ Magick, konnectors and services for example). And they can take several GB for
 the case of importing a Cozy. If needed, it is possible to configure the
 directory where they will be created via the `TMPDIR` environment variable.
 
+## Multiple CouchDB clusters
+
+With a large number of instances, a single CouchDB cluster may not be enough.
+Most cozy-stack administrators can ignore this section, but if you need this
+advanced settings, let's see how to configure it. We start with a classical
+configuration for a single cluster:
+
+```yaml
+couchdb:
+  url: http://cluster1:5984/
+```
+
+The first step is to add the new cluster, but still forces the creation on the
+first cluster for the moment:
+
+```yaml
+couchdb:
+  url: http://cluster1:5984/
+  clusters:
+    - url: http://cluster1:5984/
+      instance_creation: true
+    - url: http://cluster2:5984/
+      instance_creation: false
+```
+
+Then, we can restart the cozy-stack and every thing will work the same, except
+that the stack now knows how to use an instance on the new cluster. We can now
+use the second cluster for instance creations instead of the first:
+
+```yaml
+couchdb:
+  url: http://cluster1:5984/
+  clusters:
+    - url: http://cluster1:5984/
+      instance_creation: false
+    - url: http://cluster2:5984/
+      instance_creation: true
+```
+
```

Comment:
I would explain here what happens if multiple clusters have `instance_creation` set to true. Is this round-robin? first always wins? and if round-robin, is it reset to first cluster at each stack restart?

Reply:
It would also be nice to state here what would happen in case of cluster list change (changing url, or removing a cluster in the middle of the list).
I suspect couch cluster is stored as int representing list index so moving the list could result in unexpected behaviour like rendering instances as broken. Maybe it would be nice to add a cluster unique "name" or "slug" in the definition to identify the cluster?

Reply:
> I would explain here what happens if multiple clusters have instance_creation set to true.

:+1: I have noted that in the commit message of https://github.com/cozy/cozy-stack/pull/3367/commits/542320097172c63b8d55245f7958e100ee026a4b but I forgot it in the doc.

> Maybe it would be nice to add a cluster unique "name" or "slug" in the definition to identify the cluster?

I have hesitated about that, but it makes harder the migration from a single cluster, so I've taken the path of using the list index.